### PR TITLE
Added comment model

### PR DESCRIPTION
**Pull request by aibunny**
# Description
#8

### CODE REVIEW

**Review by Collins-Omariba — DISMISSED** [review]

File: blog/models.py [hunk 1]
```diff
@@ -38,8 +38,23 @@ def __str__(self):
...
     instance.slug = slugify(instance.title)
-pre_save.connect(auto_slug,sender=Stori)
\ No newline at end of file
+pre_save.connect(auto_slug,sender=Stori)
+
+
+class Comment(models.Model):
+    body = models.TextField()
+    Post_id = models.ForeignKey(Stori,on_delete= models.CASCADE)
+    user = models.ForeignKey(Account, on_delete= models.CASCADE)
```
Comment (sangkips) [inline blog/models.py:53]: Is it the Account we are supposed to refer here or the User?

**Review by sangkips — COMMENTED** [review]
user = models.ForeignKey(Account, on_delete= models.CASCADE). Elaborate why you are referencing Account and not User

File: blog/models.py (hunk 1 above)
Comment (Fredrick-Waihenya) [reply blog/models.py:53]: Because  the account also refers to the user model and as i understand the user should be able to have multiple accounts.

Comment (JimmyTron) [reply blog/models.py:53]: the comment is owned by the account of the user not the user

**Comment by Msdot001** [comment]
@Fredrick-Waihenya, I am currently working on the same issues and your model perfectly looks OK. I will appreciate it if you can inform me what you are currently doing on the issue in order to avoid repetition and to save time.

**Comment by Fredrick-Waihenya** [comment]
tonight after the meeting will know what we're at.

**Review by Collins-Omariba — APPROVED** [review]
LGTM

**Review by JimmyTron — APPROVED** [review]
Though u deleted migrations that could be a cause for more problems later.

**Pull request merged**